lib/client.js

```diff
@@ -189,7 +189,11 @@ class Client extends EventEmitter {
     try {
       this.driver = this._driver();
     } catch (e) {
-      const message = `Knex: run\n$ npm install ${this.driverName} --save`;
+      let driverName = this.driverName;
+      if (driverName === 'sqlite3') {
```

Comment:
I would suggest more generic solution of driverName override map or, even better, make Client instances aware of their driverName

Reply:
I introduce an alias driver name in sqlite3 index.js, I can't change the driver name by the way (is used by all tests that use testSql...).
By the way, I retest to delete the package, and run the command "npm install @vscode/sqlite3 --save" and I don't have any problem. I can't see more to do here to fix the problem in the related issue.

Reply:
I think current solution is perfect